integration/helium.md

```diff
@@ -1 +1,86 @@
+# Helium integration
 
+A HTTP Integration allows you to upload data to TTN Mapper. 
+
+The goal of TTN Mapper is to provide a map of the actual coverage of LoraWAN Hotspots. Contributors to TTN Mapper measure the performance of Hotspots in their vicinity and upload this information to the TTN Mapper website. Here the information is aggregated and shared with the TTN community.
+
+Go to https://ttnmapper.org for the global coverage map.
+
+
+## Prerequisites
+
```

Comment:
We can maybe just say: use the same format as is required by Cargo

Reply:
I will add the suggestion but I find we should provide our own specification here.